### DISCUSSION TITLE
Rinkeby faucet offline

### DISCUSSION THREAD

**Discussion by vinac-07** [smartcontractkit/full-blockchain-solidity-course-py#592] [Q&A] [answered]
could someone please send me test eth 
wallet address - 0x2aCC9394af4cf626f835C92d0DF27c857F074Cef

**Comment by cromewar** [comment, marked as answer]
Hello @vinac-07 sent to rinkeby: https://rinkeby.etherscan.io/tx/0x9650be91f4403c9b15277b96bdc81ad03ec59107a6f9121bf4d57f4204a8dd59

Additionally you can use chainlink faucet.

**Reply by vinac-07** [reply]
thank you @cromewar will try using chainlink faucet

**Reply by bvssrsv** [reply]
Cromeware, I am new to this and am still in 35th min.

Can you please fund my account on Rinkeby too?
https://rinkeby.etherscan.io/address/0x1383d125DB59CCC72Be777Df21F010cA8751490B
account: 0x1383d125DB59CCC72Be777Df21F010cA8751490B

**Reply by cromewar** [reply]
@bvssrsv sent: https://rinkeby.etherscan.io/tx/0xe804a78693d5288a859989a63e92f32d90a14b5d1f9a98d4d80b8cb7ae1e382a

**Comment by anooprh** [comment]
@cromewar I'm in need of some rinkeby tokens. Can you send some to me too, preferably equal to the amounts used in the video i.e 18.75 rETH. My address 0xFed64E72f08272D9e667ce4718167Ea67304cB41 

I got 0.01 rETH via the chainlink faucet, but I'm not able to co-relate the gas amounts in the video w.r.t 0.01 rETH and 18.75 rETH will make it easier to follow along the video.

**Reply by cromewar** [reply]
You actually don't need that much ETH as the 18.75 in the video is just as reference, you can use just 1 or 2, anyways I sent you some eth to your wallet: https://rinkeby.etherscan.io/tx/0xc96c786148b97d2e59e43fa2cb20968cfa7fc5c394959817b558643b1af96953

**Reply by anooprh** [reply]
Thank you.

**Comment by brijeshsonraj** [comment]
Hi , can some please send me some Rinkeby test ether for my account : 0x09B99A742a57eD2B0f03aE7dda894c019112f981 
Thanks in advance!

**Reply by cromewar** [reply]
Sent: https://rinkeby.etherscan.io/tx/0x3dfbd863eea4f107ebb9070eea13d65cde2cf139d3a4da163b83723e915320cb

**Reply by brijeshsonraj** [reply]
Thanks @cromewar !! :)
